samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# Oberon is required for spake2+
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
+
+# Mbedtls configuration
```

Comment:
```suggestion
# Mbed TLS configuration
```
Or maybe rather say something like `needed for spake2+`?

Reply:
Same as the above comment. Let's not refactor comments now